components/apimgt/org.wso2.carbon.apimgt.rest.api.gateway.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/gateway/v1/impl/LocalEntryApiServiceImpl.java

```diff
@@ -28,19 +28,29 @@
 import org.wso2.carbon.apimgt.gateway.InMemoryAPIDeployer;
 import org.wso2.carbon.apimgt.gateway.utils.LocalEntryServiceProxy;
 import org.wso2.carbon.apimgt.impl.APIAdminImpl;
+import org.wso2.carbon.apimgt.impl.APIConstants;
 import org.wso2.carbon.apimgt.rest.api.gateway.v1.*;
 import org.apache.cxf.jaxrs.ext.MessageContext;
 import org.wso2.carbon.apimgt.rest.api.util.utils.RestApiUtil;
 
 import javax.ws.rs.core.Response;
+import java.util.Map;
 
 public class LocalEntryApiServiceImpl implements LocalEntryApiService {
 
     private static final Log log = LogFactory.getLog(LocalEntryApiServiceImpl.class);
+    private final String SUPER_TENAT_DOMAIN = "carbon.super";
```

Comment:
Don't we have this super tenant as a constant in APIConstants? If so please point to that.

Reply:
https://github.com/wso2/carbon-apimgt/pull/9009/commits/b99a273017722380eaa25c62f86587c522498b9a